app/views/users/verify_personal_key/new.html.erb

```diff
@@ -1,21 +1,19 @@
-<div class="relative">
-  <%= form_tag create_verify_personal_key_path, id: 'verify-key', method: 'post', name: 'verify-key' do %>
-    <div class="padding-top-6 padding-x-1 tablet:padding-x-8 key-badge border border-dashed border-secondary rounded-xl">
-      <h3 class="margin-top-0 margin-bottom-2">
-        <%= t('forms.personal_key.title') %>
-      </h3>
-      <p class="margin-bottom-4">
-        <%= t('forms.personal_key.instructions') %>
-      </p>
+<%= form_tag create_verify_personal_key_path, id: 'verify-key', method: 'post', name: 'verify-key' do %>
```

Comment:
This is primarily indentation changes, easier to review with whitespace hidden:

https://github.com/18F/identity-idp/pull/5930/files?w=1